resources/views/site/blocks/showcase_multiple.blade.php

```diff
@@ -0,0 +1,72 @@
+@php
+    $header = $block->input('header');
+    $intro = $block->input('intro');
+@endphp
+
+<div id="{{ $block->input('styling') }}" class="m-showcase-multiple-block">
```

Comment:
On the off chance there are two showcase blocks with the same styling, add something to this ID that is unique to this block instance.

Reply:
Can you give an example of what that might be? It can't be dynamic because the css uses this id to apply the specific styles to the block.

Reply:
I think we're over-complicating things a bit. Taking a step back, 1) we need ids on the showcase multiple (and showcase) blocks so that we can link to them in the page navigation menu, and 2) as Trevin pointed out, since unique styling may not apply to blocks in all landing pages, we shouldn't explicitly shoehorn that in here. We can trivially apply the same style to multiple ids in the scss itself. All we need the editor to do here is write the same string in the showcase multiple's Block ID field as they did in navigation item's Link field.

Reply:
Ah! Good solved. Rather than a field where they enter the id, could the ID be determined based on the block title? Lowercasing and snake-casing it?

Reply:
That's how I'd originally implemented it, but Josh pointed out that editors may want to change the block title, and that doing so shouldn't break the style for that block.